src/helix/core/HelixSprite.hx

```diff
@@ -142,6 +147,35 @@ class HelixSprite extends FlxSprite
         }
         return y;
     }
+    
+    private function resetAcceleration():Void
+    {
+        this.acceleration.set();
+        this.angularVelocity = 0;
+    }
+
+    public function getFlxKeyArray(keys:Array<String>):Array<FlxKey>
+    {
+        var flxKeyArray = new Array<FlxKey>();
+
+        for (key in keys) {
+            var flxKey:Null<FlxKey> = FlxKey.fromString(key);
+
+            if (flxKey != null) {
+                flxKeyArray.push(flxKey);
+            }
+        }
+        return flxKeyArray;
+    }
+
+    private function processKeybinds():Void

```

Comment:
This should be called in `update`, if you still need it after changing the map to a single callback.